prisma/migrations/20230421174513_track_discord_id/migration.sql

```diff
@@ -0,0 +1,13 @@
+/*
+  Warnings:
+
+  - The primary key for the `players` table will be changed. If it partially fails, the table could be left without primary key constraint.
```

Comment:
Is it worth backing up the original table?

Reply:
This isn't destructive so it should be fine, but I backed up the db before submitting the PR